docs/administration.md

```diff
@@ -152,7 +152,7 @@ following:
 
     ```shell-session
     $ cd src
-    $ python3 manage.py migrate
+    $ [sudo -Hu <paperless_user>] python3 manage.py migrate
```

Comment:
I suppose a bare metal installer is probably more technically inclined, but this might be a good use of [code annotations](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) to highlight the optionality?

Reply:
Great idea, I can look into this later

Reply:
Love this feature, great call!

Reply:
Love this feature, great call!